Add some clarifications to the demo attack descriptions

awwad opened the pull request:
This PR adds some clarifications to the README.md section of instructions and descriptions of attacks to try out.

awwad committed:
DOC: Add some clarifications to the demo attack descriptions

vladimir-v-diaz commented on README.md:
```diff
@@ -443,8 +443,10 @@ not enough to allow arbitrary package attacks against ECUs in the vehicle.
-But what happens if, at the same time, the attacker is able to sign with an
-image-signing key for the Image Repository?
+But what happens if, at the same time, the attacker is able to get a hold of a
+high-level image-signing key trusted by the Image Repository? (Note that these
```
This might  be nitpicking, but the previous phrasing captures the case where an attacker uses an HSM (containing the offline key) to maliciously sign for images.  The attacker doesn't necessarily need to get ahold of a high-level signing key.

awwad replied:
That is a good point. I don't usually worry about distinguishing those scenarios, but it's important not to confuse people. I'll switch the wording back to "sign with" in the incoming commit. (I could say "gains access" or "gains the use of", but I think "sign with" makes sense.)

vladimir-v-diaz commented on README.md:
```diff
@@ -518,6 +538,17 @@ both repositories. This is the first time they have needed to be used since the
+that delegates would do the revocation. So, for example, if you delegate
+the ability to sign for firmware from a given vendor to the security team from
+that vendor, and that team delegates in turn to a particular release manager,
+and that release manager's YubiKey falls into the sewers, the security team
```
Oh, no.  The sewers?  Good, better than discovering that you have a vulnerable authentication device.

awwad replied:
Variety is the spice of security.

vladimir-v-diaz reviewed (commented):
LGTM.  I just had one suggestion for this pull request, as well.

awwad committed:
DOC: Clarify scenario text ("sign with" a key, not just "get")

so as not to spur thoughts like "Oh, but there's an HSM", so you
can't really "get a hold of" some key.
